CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## master / unreleased
 
+## 0.7.0-rc.0 / 2020-03-09
+
```

Comment:
Can we make it clear that this release has a bunch of breaking changes, and that we are making those changes in anticipation of a 1.0 release soon which will offer much more stability when upgrading?

Reply:
Uhm, sure but this is not the Changelog for the Cortex 0.7 release. This is just to add a new marker in the changelog and having new PRs merged that will add the changelog log entry under `master / unreleased`. I will submit a different PR for the Cortex 0.7 release (and that PR will be merged once Cortex `0.7.0` release will be out). Makes sense?